Client/Frontend/Browser/URLBarView.swift

```diff
@@ -339,7 +343,7 @@ class URLBarView: UIView {
         locationTextField = ToolbarTextField()
 
         guard let locationTextField = locationTextField else { return }
-
+        
```

Comment:
nit: whitespace

Reply:
My editor is somewhere adding these lines. hmm.